README.md

```diff
@@ -95,6 +95,7 @@ In addition to ember-cli-typescript, we make the following changes to your proje
   * **@types/rsvp** - ([npm](https://www.npmjs.com/package/@types/rsvp) | [source](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/rsvp)) - Types for [RSVP.js](https://github.com/tildeio/rsvp.js/)
   * **@types/ember-test-helpers** - ([npm](https://www.npmjs.com/package/@types/ember-test-helpers) | [source](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/ember-test-helpers)) Types for [ember-test-helpers](https://github.com/emberjs/ember-test-helpers), which arose from [RFC #232](https://github.com/emberjs/rfcs/blob/master/text/0232-simplify-qunit-testing-api.md)
   * **@types/ember-testing-helpers** - ([npm](https://www.npmjs.com/package/@types/ember-testing-helpers) | [source](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/ember-testing-helpers)) – Types for [Ember's built-in globally-available test helpers](https://github.com/emberjs/ember.js/tree/master/packages/ember-testing/lib/helpers)
+  * **@types/ember__test-helpers** - ([npm](https://www.npmjs.com/package/@types/ember__test-helpers) | [source](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/ember__test-helpers)) – Types for [ember-test-helpers](https://github.com/emberjs/ember-test-helpers) when imported as `@ember/test-helpers`.
 
```

Comment:
I am not sure that this is the best way to describe what is going on here. Because... I do not actually know what is going on here. Since this all comes from the same package that is already typed a few lines up above, I don't know why we need this definition, too. Shouldn't the other one declare both modules?

Reply:
There are a couple factors in play. One is that tools like VS Code can use import paths to provide intelligent typeahead and documentation even in plain JS files, but the apparent package name in the path needs to match up with the published `@types` package for that to work.

The other is that `ember-test-helpers` is actually no longer getting new releases — it's being published as `@ember/test-helpers` now, and that re-exports some old stuff (that's slated for deprecation, I believe) along with all the new helpers under the old `ember-test-helpers` namespace. Eventually, `@ember/test-helpers` will be the only thing left.

I think we probably want to move toward not adding `@types/ember-test-helpers` _or_ `@types/ember-testing-helpers` to new projects, but that can happen as follow on work.